test/winter_onboarding_2021/fleet_management/handlers/cab_test.clj

```diff
@@ -50,42 +50,42 @@
                               :licence-plate "Foo Licence Plate"
                               :distance-travelled 19191})
           content (views/cab cab)]
-      (is (= {:title (:name cab)
+      (is (= {:title (:cabs/name cab)
               :content content}
              (handlers/view-cab {:params {:id (str (:cabs/id cab))}}))))))
 
 (deftest list-cabs-handler
-  (testing "Should return a list of 3 rows of cabs"
-    (let [cabs (factories/create-cabs 3)]
-      (doall (map models/create cabs))
-      (with-redefs [config/get-page-size (constantly 2)]
-        (is (= 2 (count (hf/hiccup-find [:tbody :tr] (handlers/get-cabs {})))))
-        (is (not-empty (hf/hiccup-find [:#cab-next-page] (handlers/get-cabs {})))))))
+  (testing "Should return a list of 2 rows of cabs"
+    (with-redefs [config/get-page-size (constantly 2)]
+      (let [cabs (factories/create-cabs 3)
+            _ (doall (map models/create cabs))
+            output (handlers/get-cabs {})]
+        (is (= 2 (count (hf/hiccup-find [:tbody :tr] (:content output)))))
```

Comment:
Can we assert on the content here and not just the count?

Reply:
Done. Please check @neenaoffline